linux_os/guide/services/ssh/ssh_server/disable_host_auth/rule.yml

```diff
@@ -7,8 +7,10 @@ description: |-
     more secure than <tt>.rhosts</tt> authentication. However, it is
     not recommended that hosts unilaterally trust one another, even
     within an organization.
-    <br /><br />
-    To disable host-based authentication, add or correct the
+    <br />
+    The default SSH configuration disables host-based authentication.
+    <br />
+    To explicitly disable host-based authentication, add or correct the
```

Comment:
@adelton Hello, does the change above make it more clear what is the default value and the behaviour of the rule?

Do you think some more explicit is needed? For example:
`
The default SSH configuration disables host-based authentication. The appropriate value will be used by SSH if no configuration file sets a value for <tt>HostbasedAuthentication</tt>.
`

Reply:
@vojtapolasek your feedback is also welcome, :)

Reply:
I like your idea of the more explicit description. For the users it's hard to find whether they have the default settings or whether the default settings have been modified by something.

Reply:
I think I'm leaning towards making the description of default config more explicit.